### ISSUE TITLE
Latest version of the package is compiled with System.Text.Json 8.0.0 which has known vulnerabilities.

### ISSUE THREAD

**Issue by dougmorris** [dvsekhvalnov/jose-jwt#257]
It would be better compile with 8.0.5 of System.Text.Json

**Comment by dvsekhvalnov** [comment]
it's actually referencing `5.0.2` for .netstandard 2.1 only. 

but guess can bump up dependencies, why not.

**Comment by dougmorris** [comment]
![image](https://github.com/user-attachments/assets/363ec635-1d79-44ae-b31f-6c1411c0c52c)

**Comment by dvsekhvalnov** [comment]
yeah, i guess it's resolution thing, here is reference: https://github.com/dvsekhvalnov/jose-jwt/blob/master/jose-jwt/jose-jwt.csproj#L105

i'll update package on nuget, but pretty sure you can specify fixed version in your project in a mean time.

**Comment by dvsekhvalnov** [comment]
Hey @dougmorris , published pre-release, bumped up version to `9.0.0`, https://github.com/dvsekhvalnov/jose-jwt/releases/tag/v5.1.1-alpha wanna try to see if it is better?

**Comment by dougmorris** [comment]
@dvsekhvalnov tried and it worked for me thank you! 🙏

**Comment by dvsekhvalnov** [comment]
v5.1.1 released to nuget.
